lib/evmone/baseline.cpp

```diff
@@ -46,14 +53,10 @@ const uint8_t* op_jump(
 }
 
 template <size_t Len>
-inline const uint8_t* load_push(
-    ExecutionState& state, const uint8_t* code, const uint8_t* code_end) noexcept
+inline const uint8_t* load_push(ExecutionState& state, const uint8_t* code) noexcept
 {
-    // TODO: Also last full push can be ignored.
-    if (code + Len > code_end)  // Trimmed push data can be ignored.
-        return code_end;
-
     uint8_t buffer[Len];
+    // This cannot overflow code buffer because code ends with valid STOP instruction.
```

Comment:
I think here it's not relevant that it's a STOP?
```suggestion
    // This cannot overflow code buffer because code is padded with 0s.
```

Reply:
I think the original comment is more correct, as the loop is looking for STOP

Reply:
I thought the comment here is supposed to explain why `memcpy` cannot overflow.